### DISCUSSION TITLE
[Question] Whether the tutorial on the website will have multi -language support plans in the future

### DISCUSSION THREAD

**Discussion by zhicheng-ning** [rabbitmq/rabbitmq-website#1559] [General]
Whether the tutorial on the website will have multi -language support plans in the future？like Chinese？Maybe I can help it.

**Comment by michaelklishin** [comment]
Thank you for considering improving RabbitMQ docs.

We will likely keep the docs in English for the foreseeable future. Even though our team speaks at least 9 languages between the core team and platform engineering, maintaining even a couple extra
translations of all the tutorial ports is going to be very time consuming.

So it's not the initial translation that will take a lot of resources, it the continued maintenance,
even if we focus on just the tutorials.

**Reply by zhicheng-ning** [reply]
I got it, thanks for your answer.
